driver-core/src/test/functional/com/mongodb/OperationFunctionalSpecification.groovy

```diff
@@ -249,6 +263,10 @@ class OperationFunctionalSpecification extends Specification {
         def readBinding = Stub(ReadBinding) {
             getReadConnectionSource() >> connectionSource
             getReadPreference() >> readPreference
+            getSessionContext() >> Stub(SessionContext) {
+                hasActiveTransaction() >> false
+                getReadConcern() >> readConcern
```

Comment:
Plumbing the read concern down into the operation tests, now that it's required to be a part of the session context.